### PR TITLE
Fix/1602  fix responsive design and fix header height

### PR DESCRIPTION
**Pull request by Skylsmoi**
https://github.com/tracim/tracim/issues/1602

also included: https://github.com/tracim/tracim/issues/1074 (reduce header height)

### CODE REVIEW

File: functionnal_tests/cypress/integration/account/account_main_page.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ describe('account :: main_page', function () {
   })
   it('', function () {
     cy.get('#dropdownMenuButton').click()
-    cy.get('a.setting__link[href="/ui/account"]').click()
+    cy.get('a.menuprofil__dropdown__setting__link[href="/ui/account"]').click()
```
Comment (basilelegal) [inline functionnal_tests/cypress/integration/account/account_main_page.js]: You could use the opportunity to change `class` selector by `data-cy` selector

File: frontend_lib/src/css/Variable.styl [hunk 2]
```diff
@@ -30,3 +30,5 @@ min-lg = 992px
 max-lg = 1199px
 
 min-xl = 1200px
+
+headerHeight = 61px // the total header height. Used to calculate height of layout like height: calc(100vh - headerHeight)
```
Comment (basilelegal) [inline frontend_lib/src/css/Variable.styl]: Missing name and date for the comment

File: frontend/src/container/Login.jsx [hunk 3]
```diff
@@ -149,93 +148,65 @@ class Login extends React.Component {
...
+                maxLength={512}
+              />
+
+              <InputGroupText
+                parentClassName='loginpage__card__form__groupepw'
+                customClass=''
+                icon='fa-lock'
+                type='password'
+                placeHolder={props.t('Password')}
+                invalidMsg='Mot de passe invalide.'
```
Comment (basilelegal) [inline frontend/src/container/Login.jsx]: French ?

File: frontend/src/container/Login.jsx [hunk 4]
```diff
@@ -149,93 +148,65 @@ class Login extends React.Component {
...
+
+          <CardBody formClass='loginpage__card__form'>
+            <form>
+              <InputGroupText
+                parentClassName='loginpage__card__form__groupemail'
+                customClass='mb-3 mt-4'
+                icon='fa-envelope-open-o'
+                type='email'
+                placeHolder={props.t('Email Address')}
+                invalidMsg='Email invalide.'
```
Comment (basilelegal) [inline frontend/src/container/Login.jsx]: French ?

File: frontend/src/container/ForgotPassword.jsx [hunk 5]
```diff
@@ -51,62 +51,56 @@ export class ForgotPassword extends React.Component {
...
-                      maxLength={512}
-                    />
+              <InputGroupText
+                parentClassName='forgotpassword__card__body__groupemail'
+                customClass=''
+                icon='fa-envelope-open-o'
+                type='email'
+                placeHolder={props.t('Email')}
+                invalidMsg='Email invalide.'
+                // isInvalid={this.state.inputLogin.isInvalid}
```
Comment (basilelegal) [inline frontend/src/container/ForgotPassword.jsx]: is this useful to keep ?

File: frontend/src/container/Login.jsx (hunk 3 above)
Comment (lebouquetin) [reply frontend/src/container/Login.jsx]: it is also missing translation process

File: frontend/src/container/Login.jsx (hunk 4 above)
Comment (lebouquetin) [reply frontend/src/container/Login.jsx]: it is also missing translation process

File: frontend/src/container/ForgotPassword.jsx (hunk 5 above)
Comment (lebouquetin) [reply frontend/src/container/ForgotPassword.jsx]: A good habit about comment is the following one: never comment code without explanation. The explanation is the following one:

- if the code must be kept commented in the code, then there is a good reason (and it must be explained)
- if it's a comfort, then it must be removed (the developer will be able to get the code back from old commits)
- if it's an error, then it must be fixed.

**Comment by lebouquetin** [comment]
For me the MR is ok as soon as french translations are fixed (for the rest, you must agree @Skylsmoi and @basilelegal )

File: frontend/src/container/ForgotPassword.jsx (hunk 5 above)
Comment (Skylsmoi) [reply frontend/src/container/ForgotPassword.jsx]: it's invalid, I removed it

File: frontend/src/container/Login.jsx (hunk 4 above)
Comment (Skylsmoi) [reply frontend/src/container/Login.jsx]: translation process added

File: frontend/src/container/Login.jsx (hunk 3 above)
Comment (Skylsmoi) [reply frontend/src/container/Login.jsx]: translation process added

File: frontend_lib/src/css/Variable.styl (hunk 2 above)
Comment (Skylsmoi) [reply frontend_lib/src/css/Variable.styl]: added comment convention

File: functionnal_tests/cypress/integration/account/account_main_page.js (hunk 1 above)
Comment (Skylsmoi) [reply functionnal_tests/cypress/integration/account/account_main_page.js]: added data-cy and describe + it proper description

**Pull request merged**